app/src/main/java/io/github/droidkaigi/confsched2018/presentation/detail/SessionDetailActivity.kt

```diff
@@ -46,17 +56,46 @@ class SessionDetailActivity :
 
     private val pagerAdapter = SessionDetailFragmentPagerAdapter(supportFragmentManager)
 
+    private lateinit var sessions: List<Session.SpeechSession>
+
+    val firstSessionId: String by lazy {
+        intent.getStringExtra(EXTRA_SESSION_ID)
+    }
+
+    @TargetApi(Build.VERSION_CODES.LOLLIPOP)
+    private val sharedElementCallback = object : SharedElementCallback() {
```

Comment:
I have a question 🙋 
SharedElementCallback does not contain API Level 19.
Is this not crash on API Level 19?